html_css/css_flexbox.md

```diff
@@ -0,0 +1,27 @@
+### Introduction
+
+The Flexible Box Module, usually referred to as flexbox, was designed as a one-dimensional layout model, and as a method that 
+could offer space distribution between items in an interface and powerful alignment capabilities. This article gives an outline
+of the main features of flexbox, which we will be exploring in more detail in the rest of these guides.
+
+In this section, you'll get a look at how to use CSS Flex Box to make your projects more awesome!.
+
+### Learning Outcomes
+* What is CSS flex?
+* Why are flex effective for helping site layout?
+* What are the major different types of flex in use today?
+* How could you design your own website using flex?
+* How to use flex-box?
+
+### Assignment
+<div class="lesson-content__panel" markdown="1">
+1. Complete this game based tutorial for learning the [Basics of Flexible Box](https://flexboxfroggy.com/).
+2. Learn the modern CSS layouts from [this tutorial](https://internetingishard.com/html-and-css/flexbox/).
+3.Before we begin, it’s important for you to be familiar with basic [HTML](https://www.theodinproject.com/courses/html5-and-css3/lessons/html5-basics) and [CSS](https://www.theodinproject.com/courses/html5-and-css3/lessons/css3-basics). It would also be helpful to know a CSS Framework like Bootstrap. It would help you get the most from here
```

Comment:
This lesson would go into the middle of the html and css course so I think this assignment is not needed.

Reply:
But this assignment includes all the best resources about learning flex.